cmd/aspect/clean/clean.go

```diff
@@ -53,16 +61,23 @@ Workaround inconistent state:
 	Such problems are fixable and these bugs are a high priority.
 	If you ever find an incorrect incremental build, please file a bug report,
 	and only use clean as a temporary workaround.`,
-		RunE: b.Run,
+		RunE: pathutils.InvokeCmdInsideWorkspace(func(workspaceRoot string, cmd *cobra.Command, args []string) (exitErr error) {
+			bzl.SetWorkspaceRoot(workspaceRoot)
+			isInteractive := isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd())
```

Comment:
didn't we fix this already? we should honor the `--interactive` CLI flag too

Reply:
I guess there are some leftovers. I'll do a second pass on another PR to fix them all.